test/types/community/collection/count.test-d.ts

```diff
@@ -0,0 +1,14 @@
+import { expectDeprecated, expectType } from 'tsd';
+import { MongoClient } from '../../../../src/index';
+
+// test collection.countDocuments
+const client = new MongoClient('');
+const collection = client.db('test').collection('test.count');
+
+expectType<number>(await collection.countDocuments());
+expectType<number>(await collection.countDocuments({ foo: 1 }));
+expectType<number>(await collection.countDocuments({ foo: 1 }, { limit: 10 }));
+
+// Make sure count is deprecated
+
```

Comment:
this is super nitpicky, but could we get rid of the extra blank line so that it's similar in style to the other comments that describe the test?